src/diffpy/srmise/tests/test_dataclusters.py

```diff
@@ -32,3 +33,86 @@ def test___eq__():
             print(f"not-equal test failed on {attr_key}")
             assert not expected == actual
         attributes.update({attr_key: reset})
+
+    # In the set data test, we test for these cases.
+    # (1) x and y are non-empty array values, and res is positive (the most generic case)
+    # (2) x and y are non-empty array values, and res is 0 (will produce a msg that makes trivial clustering)
```

Comment:
I would not raise errors or warnings for this.  It is just the identity (gives back the input unchanged) which is not that useful but not anything invalide.   Please make an issue to address this in the docs, but not in the code.